tests/console/orphaned_packages_check.pm

```diff
@@ -62,9 +62,7 @@ sub compare_orphans_lists {
 sub run {
     select_console 'root-console';
 
-    record_info('Upgraded?',
-        'Has the SUT been upgraded or installed from LIVECD? Both can possibly cause orphans',
-        result => (is_offline_upgrade_or_livecd) ? 'ok' : 'fail');
+    record_info((is_offline_upgrade_or_livecd) ? 'Upgrade/LiveCD' : 'No upgrade/LiveCD');
```

Comment:
Looks good, but I would leave the hint in case SUT has been upgraded that this might cause orphan packages, otherwise it's not clear why we has this info box. Other than that LGTM.

Reply:
yes, thanks!